app/views/home/sell.html.haml

```diff
@@ -44,7 +44,6 @@
 
         %p
           = t :sell_embed
-          %a{href: "hello@openfoodnetwork.org".reverse, target: '_blank', mailto: true}
-            = t :sell_ask_services
+          = mail_to ContentConfig.footer_email, t(:sell_ask_services)
```

Comment:
Is the `reverse` missing here? We use it to obfuscate the email address. The `mailto` attribute triggers some javascript to un-reverse it again for the user.